### ISSUE TITLE
freedom and privacy respecting communication options for LBRY

### ISSUE THREAD

**Issue by ITwrx** [lbryio/lbry#1963]
hi,
Please consider using freedom and privacy respecting software for the official LBRY project communication methods. I can understand needing to reach people on proprietary, surveillance platforms (aka digital plantations/auction blocks) but LBRY should set a good example (and stay aligned with what i presume is it's own desired public image) by promoting the use of software that respects peoples' rights as it's primary and official communication channels.

github? how about gitea or gitlab(open core. at least) or similar.
twitter? how about pleroma(lightweight) or mastadon?
facebook? what about minds.com, or maybe one of the self hosted options(these are less mature).
reddit? maybe just add native comments (not disqus or similar!) to the lbry news posts.
discord? rocketchat or similar.
telegram? rocketchat, xmpp, irc.
instagram? pixelfed.

Right now, it looks like the only way someone could communicate with the LBRY project or community without being subjugated by enslaveWare is to use the LBRY Discourse forum, but that is advertised as for developers only. 

thanks

**Comment by nikooo777** [comment]
Hi @ITwrx 

Thank you for your feedback and suggestions.
I'm just an employee and I do not by any means take any final decisions, but I'd like to try and explain why we're doing one thing and not the other.

We are a limited team, a quite "small" one, most of us are coders and spend most of the day writing code for LBRY, some of us are community oriented and will engage with the people to offer them support and to attract them to LBRY.

We need to obviously optimize the time we spend engaging with people, and to do so we use platforms that the majority of the people will be.

We'd love to support all the alternatives of the world, but it's simply impossible with so little time and resources available. Hopefully one day we'll be able to do just that, but until then, we should better focus on providing a great experience to the user and an easy way of communicating with us through commonly used channels.

I hope this addresses some of your concerns,
though those are just my 2 cents.

BR,
Niko

**Comment by ITwrx** [comment]
@nikooo777 
thanks for the response.
i'm not suggesting that people contributing to the code should stop their work and set up alternatives. i only suggest that **the project** might consider that right now people are forced to use these subjugating platforms if they want to join in any official discussion about LBRY. 

also, some of these suggestions require almost zero set up and don't have to be self hosted. Or maybe just open up the forum for anyone, instead of just devs? Why be so picky when you can just have categories in the forum like what already appears to exist. Maybe LBRY doesn't even care if anyone uses the forum currently and the way it's presented just needs to be updated?

I receive the LBRY newsletter and have been impressed by the seeming progress and organization and have promoted it to people. i finally had time to look to see if there might be something i could get involved with and i was distressed to find the currently offered options. I didn't want to just leave without saying anything in case anyone wanted to add at least one freedom friendly option/alteration to the roadmap.

while the availability of email is nice of the project to include, it does not allow one to enter into the broader discussion with the whole community. 

i'll leave this issue open for LBRY to close if they decide that the current situation is fine or it gets resolved in some way.

good luck.

**Comment by ITwrx** [comment]
btw, i have a fairly new pleroma instance that i would be glad to send invitations for (it's currently invitation only until time/features allow for moderation) to any LBRY rep(s) that wanted to use it. Just provide any email address(es) to send the invite(s) to. Otherwise, you may wish to choose from more established fediverse instances.

**Comment by ITwrx** [comment]
zulip might be useful: https://zulipchat.com

**Comment by kauffj** [comment]
@ITwrx I appreciate your passion for open-source and free technologies. Everyone at LBRY cares a lot about these things.

However, we're also looking to build a business. To do that we need to acquire users that don't care about these things. And to get them, we need to go where they are.

What we can and will look at is the feasibility of adding some of these to the places we maintain an identity.

I've opened an internal ticket to look at posting on Mastodon on a regular basis. I also opened https://github.com/lbryio/lbry.io/issues/974 to look at adding blog commenting via a non-Disqus (or similar) technology.

The rest of these we're not going to pursue for now.

I'm going to close this since it's not really appropriate for this repo anyway.
